kano_updater/scenarios.py

```diff
@@ -278,6 +279,11 @@ def beta_132_to_beta_133(self):
 
     def beta_133_to_beta_134(self):
         pass
-    
+
     def beta_134_to_beta_200(self):
-        pass
+        if is_installed('kano-character-cli'):
+            cmd = 'kano-character-cli -c "Judoka_Base" "Hair_Black" "Skin_Orange" -r -s'
+            run_cmd_log(cmd)
+        else:
+            logger.warn(
```

Comment:
This should happen in the Post Update step. If `kano-character-cli` is missing, it probably means that there is something wrong with the update. Should I bump this to error?